### ISSUE TITLE
External service dependencies?

### ISSUE THREAD

**Issue by gdevenyi** [Worklenz/worklenz#12]
The worklenz backend won't launch without S3 URL/SECRETS being defined, is S3 a requirement for the file storage backend, or can it use a local filesystem?

There's no documentation on can find on configuration of these components.

**Comment by kalinga777** [comment]
@gdevenyi yes it's having a dependency on S3. We will create an issue on this

**Comment by icharge** [comment]
It would support more storage method

**Comment by kalinga777** [comment]
Currently it supports both minio and S3
